### DISCUSSION TITLE
ArchLinux User Repository package status

### DISCUSSION THREAD

**Discussion by AnatolyRugalev** [helmfile/helmfile#278] [General]
Hi, @mumoshu 

I am a maintainer of [helmfile AUR package](https://aur.archlinux.org/packages/kubernetes-helmfile-bin), and I see that you guys have [community/helmfile](https://archlinux.org/packages/community/x86_64/helmfile/) now, making my package redundant. However, it seems like this package is stale because of the repo switch. Any plans to switch `community/helmfile` to the new source? If so, I will gladly deprecate my package and steer users to the official one.

Thanks

**Comment by mumoshu** [comment]
@AnatolyRugalev Hey! Thanks for maintaining the package. Unfortunately, we don't own `community/helmfile`, so I'm not sure what to do.
Perhaps you could contact the pkg owner for further steps, or make yours the representative package, perhaps by featuring it in helmfile's docs? WDYT?

**Comment by AnatolyRugalev** [comment]
Oh, I thought that was something officially distributed by you. I have sent a notification to the maintainer (@shibumi), let's see

**Comment by shibumi** [comment]
Oh, I totally missed the new helmfile location. Thank you for the notification. I will change the repo URL as fast as possible and update it :)

**Comment by shibumi** [comment]
Updated the new package. Thanks

**Comment by AnatolyRugalev** [comment]
I just deprecated AUR package. A README file needs an update now: https://github.com/helmfile/helmfile/pull/282
